src/User.java

```diff
@@ -0,0 +1,36 @@
+public class User {
+    private String login;
+    private String password;
+    private boolean isLoggedIn;
+
+    public User(String login, String password) {
+        this.login = login;
+        this.password = password;
+        this.isLoggedIn = false;
+    }
+
+    public boolean isLoggedIn() {
```

Comment:
mozesz to usunac

Reply:
to ciekawe jak uzyskamy stan użytkownika w przyszlosci.